addons/account/models/account_move.py

```diff
@@ -5249,27 +5249,11 @@ def _add_lines_to_exchange_difference_vals(lines, exchange_diff_move_vals):
                     continue
 
                 sequence = len(exchange_diff_move_vals['line_ids'])
+                exchange_vals_0 = self._prepare_exchange_vals_0(line, sequence)
```

Comment:
```suggestion
                exchange_vals_0 = self._prepare_exchange_vals_0(line, sequence, exchange_line_account)
```
Although not used, I think you could pass the exchange, just in case some customization needs it.